src/tests/__tags.html

```diff
@@ -5,44 +5,47 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link href="../lib/eccentric.css" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
This is a Test file. Do not write this font-awesome import statement in the HTML Test module. 
Just add the CSS Code given below in the `eccentric.scss` and `eccentric.css` module and then remove the `link` from the HTML Document.

```CSS
@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css');
```